PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx

```diff
@@ -214,6 +220,8 @@ struct HfCandidateSelectorDplusToPiKPi {
       auto trackNeg = candidate.prong1_as<TracksSel>();  // negative daughter (positive for the antiparticles)
       auto trackPos2 = candidate.prong2_as<TracksSel>(); // positive daughter (negative for the antiparticles)
 
+      LOG(info) << "Combined PID = " << trackPos1.tpctofNSigmaPi();
```

Comment:
I just noticed I forgot to remove this, I will do it when I push the next commit with the PR comments.